src/test/java/io/jenkins/plugins/analysis/core/charts/CompositeResultTest.java

```diff
@@ -89,11 +89,40 @@ void shouldCreatePriorityChartForJobAndMultipleActions() {
     }
 
     private AnalysisBuildResult createResult(final int high, final int normal, final int low, final int number) {
-        AnalysisBuildResult buildResult = mock(AnalysisBuildResult.class);
+        AnalysisBuildResult buildResult = createBuildResultMock(number);
 
         when(buildResult.getTotalSizeOf(Severity.WARNING_HIGH)).thenReturn(high);
         when(buildResult.getTotalSizeOf(Severity.WARNING_NORMAL)).thenReturn(normal);
         when(buildResult.getTotalSizeOf(Severity.WARNING_LOW)).thenReturn(low);
+        when(buildResult.getTotalSize()).thenReturn(low + normal + high);
+
+        return buildResult;
+    }
+
+    private AnalysisBuildResult createResultWithNewIssues(final int high, final int normal, final int low,
+            final int number) {
+        AnalysisBuildResult buildResult = createBuildResultMock(number);
+
+        when(buildResult.getNewSizeOf(Severity.WARNING_HIGH)).thenReturn(high);
+        when(buildResult.getNewSizeOf(Severity.WARNING_NORMAL)).thenReturn(normal);
+        when(buildResult.getNewSizeOf(Severity.WARNING_LOW)).thenReturn(low);
+        when(buildResult.getNewSize()).thenReturn(low + normal + high);
+
+        return buildResult;
+    }
+
+    private AnalysisBuildResult createResultWithFixedIssues(final int fixedIssues, final int number) {
+        AnalysisBuildResult buildResult = createBuildResultMock(number);
+
+        when(buildResult.getFixedSize()).thenReturn(fixedIssues);
+
+        AnalysisBuild build = createBuild(number);
+        when(buildResult.getBuild()).thenReturn(build);
+        return buildResult;
+    }
+
+    private AnalysisBuildResult createBuildResultMock(final int number) {
```

Comment:
Actually this is a stub, see https://martinfowler.com/articles/mocksArentStubs.html